{"name": "test_loginFailMethod", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <AppiumFramework.tests.LoginPageTest.LoginTest testMethod=test_loginFailMethod>\n\n    @pytest.mark.run(order=2)\n    def test_loginFailMethod(self):\n>       self.gt.verifyLoginScreen()\n\nLoginPageTest.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <AppiumFramework.pages.LoginPage.LoginPageTest object at 0x03D4DA70>\n\n    def verifyLoginScreen(self):\n        loginScreen = self.isDisplayed(self._username, \"id\")\n>       assert loginScreen == True\nE       AssertionError\n\n..\\pages\\LoginPage.py:45: AssertionError"}, "attachments": [{"name": "id", "source": "fdf9592b-ca2b-40fd-a6b8-40f97d370899-attachment.png", "type": "image/png"}, {"name": "id", "source": "14edbb06-f1d3-4969-b3e0-0652ad0de9a7-attachment.png", "type": "image/png"}, {"name": "log", "source": "135c45d0-ddff-44dc-8037-84c7f194900f-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "63071cbd-cdd6-4e5b-afd5-dcc7d14fd98c-attachment.txt", "type": "text/plain"}], "start": 1599738259547, "stop": 1599738286275, "uuid": "9d72df8e-fdc7-4939-8294-349a4bc1f5be", "historyId": "aeb6c84858cb897d59574054d46d09e3", "testCaseId": "1512c76b4b2d60f059541c0f73c45353", "fullName": "TestSuite.LoginTest#test_loginFailMethod", "labels": [{"name": "suite", "value": "TestSuite"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "LP-00DBDF5DBA60"}, {"name": "thread", "value": "19328-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSuite"}]}